conf/ocsci/osd_cluster.yaml

```diff
@@ -0,0 +1,3 @@
+---
+ENV_DATA:
+  platform: 'osd'
```

Comment:
not to be confused with ceph osd which is used in other places in the script, we should name it "OpenshiftDedicated"

Reply:
Agreed and modified!